### ISSUE TITLE
PM rejection

### ISSUE THREAD

**Issue by MaribelHearn** [po-devs/po-web#129]
Would be nice if you could reject PMs just like on the desktop client.

**Comment by danimesq** [comment]
I agree. The WebClient needs be the most paired possibly with Android/Desktop versions.

**Comment by sulcata** [comment]
Do you mean ignoring? Right now you can do that by right clicking the player list.

**Comment by MaribelHearn** [comment]
No, that's targetted towards a single user, and also ignores their posts on channels. I mean the global PM rejection that blocks _any_ incoming PM rather than PMs from a single user.

**Comment by coyotte508** [comment]
Is it really needed though?

PMs on the browser are a lot less annoying than in the client or the android client.

**Comment by danimesq** [comment]
@coyotte508 do you think in a possible universal app? The web power is for this.

**Comment by coyotte508** [comment]
in an universal app, it's different. It's **not now**. It's a theorical future. Show me one done and we'll discuss.

**Comment by danimesq** [comment]
I believe in javascript power and in the fact we can build apps with a unique code, releasing a single version for various platforms. I don't believe its a theorical future, but a unavoidable. But ok, when I build the Android/Desktop build, lets discuss it.
